diff --git a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
index c259beb..cb4ab10 100644
--- a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
@@ -105,6 +105,7 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
     <MainNav condensed={condensed}>
       <NavBrand
         as={RouterNavLink}
+        to="/content-manager"
         workplace={formatMessage({
           id: 'app.components.LeftMenu.navbrand.workplace',
           defaultMessage: 'Workplace',
diff --git a/node_modules/@strapi/admin/admin/src/content-manager/pages/App/LeftMenu/index.js b/node_modules/@strapi/admin/admin/src/content-manager/pages/App/LeftMenu/index.js
index 9e6f8a7..befec42 100644
--- a/node_modules/@strapi/admin/admin/src/content-manager/pages/App/LeftMenu/index.js
+++ b/node_modules/@strapi/admin/admin/src/content-manager/pages/App/LeftMenu/index.js
@@ -50,15 +50,6 @@ const LeftMenu = () => {
           searchable: true,
           links: collectionTypeLinks,
         },
-        {
-          id: 'singleTypes',
-          title: {
-            id: getTrad('components.LeftMenu.single-types'),
-            defaultMessage: 'Single Types',
-          },
-          searchable: true,
-          links: singleTypeLinks,
-        },
       ].map((section) => ({
         ...section,
         links: section.links
diff --git a/node_modules/@strapi/admin/admin/src/hooks/useMenu/utils/getGeneralLinks.js b/node_modules/@strapi/admin/admin/src/hooks/useMenu/utils/getGeneralLinks.js
index fac6937..1fe141f 100644
--- a/node_modules/@strapi/admin/admin/src/hooks/useMenu/utils/getGeneralLinks.js
+++ b/node_modules/@strapi/admin/admin/src/hooks/useMenu/utils/getGeneralLinks.js
@@ -2,7 +2,13 @@ import cloneDeep from 'lodash/cloneDeep';

 import checkPermissions from './checkPermissions';

+//only super admins should have this setting
+const UPDATE_PROJECT_SETTINGS = 'admin::project-settings.update';
+
 const getGeneralLinks = async (userPermissions, generalSectionRawLinks, shouldUpdateStrapi) => {
+  if (!userPermissions.some((permission) => permission.action ===UPDATE_PROJECT_SETTINGS)){
+    return [];
+  }
   const generalSectionPermissionsPromises = checkPermissions(
     userPermissions,
     generalSectionRawLinks
